modules/cli/src/main/java/org/apache/ignite/internal/cli/call/cluster/status/ClusterState.java

```diff
@@ -18,11 +18,12 @@
 package org.apache.ignite.internal.cli.call.cluster.status;
 
 import java.util.List;
+import org.apache.ignite.rest.client.model.ClusterStatus;
 
 /**
  * Class that represents the cluster status.
  */
-public class ClusterStatus {
+public class ClusterState {
```

Comment:
We already have an entity called 'cluster state' in the CMG domain. This one is a different entity and yet it's called identically. Could a distinct name be devised? Something like ClusterRuntimeState, maybe.